lib/TH/generic/THVectorDispatch.c

```diff
@@ -11,7 +10,7 @@
  *    which SIMD extension a given implementation uses
  * 3. A dispatch stub, which is what is actually called by clients, that simply wraps the dispatch pointer.
  */
-
+#ifndef TH_GENERIC_NO_MATH
```

Comment:
Right, it was merge atrifact between my working branch cpu_half and half_type used for this PR ... 
Fixed it now.